ecosystem/platform/server/app/components/header_component.html.erb

```diff
@@ -55,7 +55,9 @@
       </div>
     </div>
   <% else %>
-    <a title="Sign in" href="<%= new_user_session_path %>" class="h-8 font-mono font-normal text-sm flex items-center justify-center leading-none py-1 px-8 rounded-lg border border-teal-400 text-teal-300 ring-teal-400 hover:ring-1 hover:bg-neutral-800 hover:text-teal-300 uppercase z-20">Sign in</a>
+    <%= render ButtonComponent.new(href: new_user_session_path, title: 'Sign in', size: :small, scheme: :secondary, class: 'z-10') do %>
```

Comment:
I see it already had a z-index set, but why is it needed?

Reply:
This is due to the menu now having `position:absolute;` to center on larger displays, which resulted in covering this button and not being clickable.